conf/lookup/BCD-Port.csv

```diff
@@ -15520,4 +15518,101 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
-100518,ES,3,ES2,,Santander Ferry Port,Santander Ferry Port,0,3,NULL,0,1
\ No newline at end of file
+100518,ES,3,ES2,,Santander Ferry Port,Santander Ferry Port,0,3,NULL,0,1
+200000,UK,3,ARD,,Ardrossan - ARD,Ardrossan - ARD,,1,NULL,0,
+200001,UK,3,ASD,,Ashford - ASD,Ashford - ASD,,1,NULL,0,
+200002,UK,3,AVO,Portishead Royal Portbury Dock,Avonmouth - AVO,Avonmouth - AVO,,1,NULL,0,
+200003,UK,3,AYR,Girvan,Ayr - AYR,Ayr - AYR,,1,NULL,0,
+200004,UK,3,BAR,Millom,Barrow-In-Furness - BAR,Barrow-In-Furness - BAR,,1,NULL,0,
+200005,UK,3,BCC,,Beccles Airport - BCC,Beccles Airport - BCC,,1,NULL,0,
+200006,UK,3,BLY,Amble Berwick,Blyth - BLY,Blyth - BLY,,1,NULL,0,
+200007,UK,2,BOH,,Bournemouth (Hurn) Airport - BOH,Bournemouth (Hurn) Airport - BOH,,1,NULL,0,
+200008,UK,3,BOS,,Boston - BOS,Boston - BOS,,1,NULL,0,
+200009,UK,3,BWK,,Newcastle (North Sheilds Dock) - BWK,Newcastle (North Sheilds Dock) - BWK,,1,NULL,0,
+200010,UK,3,BZN,,Brize Norton - BZN,Brize Norton - BZN,,1,NULL,0,
+200011,UK,3,CDF,,Barry Dock (Cardiff) - CDF,Barry Dock (Cardiff) - CDF,,1,NULL,0,
+200012,UK,3,COL,Brightlingsea Burnham-On-Crouch Fingringhoe Maldon Rowhedge Wivenhoe,Colchester - COL,Colchester - COL,,1,NULL,0,
+200013,UK,3,COW,,Cowes - COW,Cowes - COW,,1,NULL,0,
+200014,UK,3,DAG,,Dagenham - DAG,Dagenham - DAG,,1,NULL,0,
+200015,UK,3,DEU,,Dover / Eurotunnel - DEU,Dover / Eurotunnel - DEU,,1,NULL,0,
+200016,UK,3,DEU,,Eurotunnel / Dover - DEU,Eurotunnel / Dover - DEU,,1,NULL,0,
+200017,UK,3,DGS,Castletown Peel Ramsey,Douglas - DGS,Douglas - DGS,,1,NULL,0,
+200018,UK,3,DUN,Arbroath Perth Tayport,Dundee - DUN,Dundee - DUN,,1,NULL,0,
+200019,UK,3,DVY,,Daventry - DVY,Daventry - DVY,,1,NULL,0,
+200020,UK,3,ELL,,Ellesmere Port - ELL,Ellesmere Port - ELL,,1,NULL,0,
+200021,UK,3,EUT,,Eurotunnel - EUT,Eurotunnel - EUT,,1,NULL,0,
+200022,UK,3,EXE,Exmouth Topsham,Exeter - EXE,Exeter - EXE,,1,NULL,0,
+200023,UK,3,FAL,Penryn Truro,Falmouth - FAL,Falmouth - FAL,,1,NULL,0,
+200024,UK,3,FIS,,Fishguard - FIS,Fishguard - FIS,,1,NULL,0,
+200025,UK,3,FLE,,Fleetwood - FLE,Fleetwood - FLE,,1,NULL,0,
+200026,UK,3,FLT,,Feltwell - FLT,Feltwell - FLT,,1,NULL,0,
+200027,UK,3,FOL,,Rye (Folkestone) - FOL,Rye (Folkestone) - FOL,,1,NULL,0,
+200028,UK,3,FOY,Charlestown,Fowey - FOY,Fowey - FOY,,1,NULL,0,
+200029,UK,3,FRB,Buckie Macduff,Fraserburgh - FRB,Fraserburgh - FRB,,1,NULL,0,
+200030,UK,3,GOO,Howdendyke Selby,Goole - GOO,Goole - GOO,,1,NULL,0,
+200031,UK,3,GRG,Braefoot Bay Burntisland Inverkeithing Methil,Grangemouth - GRG,Grangemouth - GRG,,1,NULL,0,
+200032,UK,3,GRI,,Grimsby - GRI,Grimsby - GRI,,1,NULL,0,
+200033,UK,3,GRK,Annan Barcaldine Campbeltown Faslane FinnartFurnace Garlieston Hunterston Islay Oban Palnackie Stranraer,Greenock - GRK,Greenock - GRK,,1,NULL,0,
+200034,UK,3,GSY,,St Peter Port (Guernsey) - GSY,St Peter Port (Guernsey) - GSY,,1,NULL,0,
+200035,UK,3,GTY,,Great Yarmouth - GTY,Great Yarmouth - GTY,,1,NULL,0,
+200036,UK,3,HEY,Lancaster,Heysham - HEY,Heysham - HEY,,1,NULL,0,

```

Comment:
Missing Glasson Dock

(Not on the list sent over over but is here:
https://www.gov.uk/government/publications/uk-trade-tariff-freight-location-codes/freight-location-codes#uk-airports)